lib/mayaUsd/python/wrapMeshUtil.cpp

```diff
@@ -59,7 +59,7 @@ _GetMeshNormals(const std::string& meshDagPath)
         return make_tuple(normalsArray, interpolation);
     }
 
-    UsdMayaMeshUtil::GetMeshNormals(meshObj, &normalsArray, &interpolation);
+    UsdMayaMeshUtil::getMeshNormals(meshObj, &normalsArray, &interpolation);
```

Comment:
There's only one function with Python bindings here and it's a write-related utility method, so this file should probably be called `wrapMeshWriteUtils.cpp`

Reply:
see 1e492f25f45430e9b5d94c846baf5fc444e37519